vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/exporter/AzureDevOpsExporter.java

```diff
@@ -46,40 +52,59 @@ public AzureDevOpsExporter(AzureDevOpsExporterOptions options, AzureDevOpsFacade
 
     public void exportResults() throws IOException
     {
+        Map<Integer, Scenario> exportedScenarios = new HashMap<>();
+
         for (Story story : OutputReader.readStoriesFromJsons(options.getJsonResultsDirectory()))
         {
             LOGGER.atInfo().addArgument(story::getPath).log("Exporting scenarios from {} story");
 
-            story.getFoldedScenarios().forEach(scenario -> exportScenario(story.getPath(), scenario));
+            story.getFoldedScenarios().stream()
+                                      .map(scenario -> exportScenario(story.getPath(), scenario))
```

Comment:
.filter(s -> {
if (scenario.hasMetaWithName("azure-devops.skip-export"))
{
    LOGGER.atInfo().addArgument(scenario::getTitle).log("Skip export of {} scenario");
    return false
}
return true;
}

Reply:
can you please explain advantage of this code change? I would use such filtering if there is only one place that might return optional empty from the exportScenario method, but there is also optional empty that gets returned from catch block